etc/ubsan.blacklist

```diff
@@ -0,0 +1,2 @@
+fun:_ZStaNRSt13_Ios_FmtflagsS_
```

Comment:
Could you add a comment or a link here? The SO post or that other one is fine. Just something so we don't puzzle over it a year from now.